### ISSUE TITLE
RP - Healing reduced significantly

### ISSUE THREAD

**Issue by bulde** [WarEmu/WarBugs#12344]
Healing on RunePriest randomly being diminished significantly (almost half!).  
I first noticed this while in a scenario.  After the scenario ended the issue persisted, so it was not just scenario related. 

*Rune of Breaking is not on.
*Divine Fury tactic not being used. 
*No heal debuff present. 

This extremely annoying issue (game breaking for a healer) started happening mid-month September 2018.  It has occurred on three different occasions now.  Below are my findings.  

Expected behavior and actual behavior:
I followed my combat log monitoring every healing ability available to the RP and all fell short of what the tool tip indicated.  Example:  Tool tip says Grungi's Gift heals for 575, actual heal was for 275.  

Steps to reproduce the problem:
Example:  Tool tip says Grungi's Gift heals for 575, actual heal was for 275.  

The only way I have been able to resolve the issue is by exiting the game completely and then entering again.  

I seem to first notice it in scenarios.  I have a suspicion that it may be occurring after switching Rune of Breaking on, then off again.  It seems as if, even though Rune of Breaking is off, it doesn't seem to be recognizing this.  I even checked my Intelligence vs Willpower to ensure it was toggling on / off and confirmed it was.  I had Rune of Breaking off, and I'm certain it was off because I did a .getstats and I had over 1000 Willpower.  There is no way that Grungi's gift should be healing for only 275 to 300 with that much Willpower.  After logging out and back in and the issues corrected, the amount returns back to what is indicated in the tool tip.  This test has been performed multiple times all in varying locations, but mostly in Altdorf. 

*** PLEASE make this an immediate priority!!   It is class and role breaking.  If I can't log in and do the fundamental task that my class is meant to do (Heal), then what is the point in even logging in? ***

**Comment by sioding** [comment]
Hey @bulde  - thanks for the report. Are you able to reproduce it and bring in some screenshots? This would be helpful. Nevertheless - i will try to get a dev looking into it.

**Comment by bulde** [comment]
Submitting screenshots: 

![bulde_015](https://user-images.githubusercontent.com/43435703/46110462-4c632580-c1a9-11e8-9a59-17900de2a937.jpg)

**Comment by bulde** [comment]
![bulde_016](https://user-images.githubusercontent.com/43435703/46110500-6ef53e80-c1a9-11e8-8834-7679b45dbd7a.jpg)

**Comment by bulde** [comment]
As you can see in my screenshots, I have over 1000 Willpower, no heal debuffs on me, not using Divine Fury tactic, and Rune of Breaking is not toggled on.  This is obviously a bug and needs to be fixed.  

One thing I have noticed, is this bug tends to occur after I have toggled Rune of Breaking on and off.  
Unlike most Runepriests, I use that ability a lot.  I switch between damage and healing more often than most.  So since it was toggled off, I went ahead and toggled it on to see if the healing would still be 302, but that wasn't  the issue.  It dropped down to 208 after turning on Rune of Breaking, so that eliminates the idea of it just being a visual bug of having Rune of Breaking turned off visually, but its really turned on.  Not the case.  It was definitely turned off during my testing.  So why then do I keep randomly have this happen?

**Comment by bulde** [comment]
...oh and for clarity.  When I have over 1000 Willpower, Grungi's Gift should be healing for well OVER what is represented in the tool tip.  So instead of healing for 521 like the tooltip indicates, it should be healing for 600+ easy.  Instead, its healing for half that.  :(

**Comment by Grognir** [comment]
Hi Bulbe, after some research here, and as my asumption were different, I did not found your post a created a new Ticket https://github.com/WarEmu/WarBugs/issues/12463. But once finished and looking at all the list, I have just seen your message.

I am Grognur IG, Rune priest who also experienced your issue several times, and sometimes twice an evening. My thoughts were that some kind of permanent outgoing debuff heal remains on our character like the perma snare issue we had by the past.

But I must confess that I regularly switch the rune of breaking on and off, and it did not come to my mind that it could be another explaination possible.

Do you have some news or other feeling about this issue?

**Comment by bulde** [comment]
Hi Grognir, thanks for the posting and helping to address the issue.  I did think about that as well, the idea that it could be a lingering outgoing heal debuff like perma snare.  If that is the case, it would have to have been introduced in a newer patch because I didn't always have this issue, not until I starting using Rune of Breaking more often.  That is why I think it is linked to Rune of Breaking.  However, it could just be coincedence too, that perhaps I started using Rune of Breaking often, around the same time a patch that may have introduced a lingering heal debuff occurred.  Still not sure....but wish this would get addressed ASAP!  It prevents me from playing my class to its greatest potential!  Which in return, makes me not want to log in and play...  :(

**Comment by bulde** [comment]
Respectfully, can you please fix my class before introducing new content into the game... i.e. Black Crag scenario.  I've still not heard anything back on this and its rather frustrating. Everything you need to know is explained in great detail above.  If you need further detail, please advise.

**Comment by sioding** [comment]
OK - i will now lock the conversation. With all the respec back to you - it´s a really good report and i am happy to have such things here, but maybe you have seen that more than 1200 open issues are out and introducing new content doesn´t mean we are not working on problems. Be sure this one is on the list but not with the highest prio.

**Comment by sioding** [comment]
Same report: #12567

**Comment by sioding** [comment]
unlocked - i give it a try again.

**Comment by XmatXergon** [comment]
Expected behavior and actual behavior:
Any heal debuff should not be permanent.
Some abilities or combinations of abilities with tactics makes permanent heal debuff on character, only relog fix it.

Steps to reproduce the problem:
I would try stack heal debuffs from Sorc, Magus, SH, WE and see if they will make permanent heal debuff.

My assumptions goes on WE most likely because only them can apply Outgoing Heal debuff.

Screenshots/Videos or archive.org evidences:
![xergoam_019](https://user-images.githubusercontent.com/37849311/48815248-1ae87f80-ed3e-11e8-87e5-84e39b799dc6.jpg)

no tactics still perma heal debuff
![xergoam_020](https://user-images.githubusercontent.com/37849311/48815251-1c19ac80-ed3e-11e8-8b6c-0e9ab1d59edc.jpg)

fixed after relog
![xergoam_021](https://user-images.githubusercontent.com/37849311/48815254-1de37000-ed3e-11e8-9a93-b5e0af964c60.jpg)

**Comment by Minipuce** [comment]
Hello,
I just had again this bug of permanent heal debuff, 2 minutes ago.
It happened in the scenario logrin's forge, just after a fight when i was alone against the witch eleves Thankyouladies and Hashdriveway who were assisting themselves (perhaps a clue on the issue ? 2 witch elves with the debuff heal ?).
My getstats showed "outgoing heal percent x0.5" even out of the scenario.
![minipuce_145](https://user-images.githubusercontent.com/39655844/49719802-7c757d00-fc5e-11e8-97de-0fb160fd3c3d.jpg)

**Comment by grunbag** [comment]
It still happen ?

**Comment by Grognir** [comment]
For my part, I have not noticed this heal debuff for months.

**Comment by grunbag** [comment]
Ok closing as I think it has been fixed already
